AdblockPlusSafari.xcodeproj/project.pbxproj

```diff
@@ -804,6 +845,36 @@
 			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-AdblockPlusSafariTests/Pods-AdblockPlusSafariTests-resources.sh\"\n";
 			showEnvVarsInLog = 0;
 		};
+		83C19D903DD1D9F927CBD253 /* Copy Pods Resources */ = {
```

Comment:
So, how I interpret this is, XCode got upgrade in the mean time and there are some pods related changes happening because of that? Could we put these in a separate commit? No need for an issue, just something like: "Noissue - XCode project file updates" would do. Then I can merge that one first.